dwarf-rust-experimental/rust/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "dwarf-rust"
+version = "0.1.0"
+description = "WebAssembly DWARF parser and location resolver"
+authors = ["Ingvar Stepanyan <rreverser@google.com>", "Hiram Chirino <hiram@hiramchirino.com>"]
```

Comment:
?

Reply:
> ?

fix the name ?

Reply:
yeah, well, not sure where this project setup is coming from :sweat_smile: